src/pipdeptree/_cli.py

```diff
@@ -60,6 +61,11 @@ def build_parser() -> ArgumentParser:
             " it can't."
         ),
     )
+    select.add_argument(
+        "--path",
+        help="comma separated list of paths used to restrict where packages should be looked for",
+        action="append",
```

Comment:
When choosing either to have a comma-separated list of paths or having users repeat `--path`, I went for the latter.

Even though our other options follow the comma-separated list approach to multiple values, I felt that having something like:

```sh
pipdeptree --paths /home/anon/site-dir-1,/home/anon/site-dir-2
```

Seems less readable when compared to:

```sh
pipdeptree --path /home/anon/site-dir-1 --path /home/anon/site-dir-2
```

... then again maybe it makes sense to stick with comma-separated lists because for consistency? Open to either one.